hw/fpga/scripts/xilinx_generate_clk_wizard.tcl

```diff
@@ -0,0 +1,50 @@
+
+# Define design macros
+set design_name                 xilinx_clk_wizard
+set in_clk_freq_hz              125000000
+set out_clk_freq_hz             20000000
```

Comment:
same, for format reasons better to define it as 20.0 and then calculate it with expressions in tcl